File: test/fixtures/python/bash_redirect.py
import requests

headers = {
    'X-Requested-With': 'curl',
    'Content-Type': 'text/xml',
}

params = {
    'action': 'create',
    'title': 'UnixRecord',
    'username': 'root',
    'password': 'abc123',
    'ips': '10.10.10.10',
}

with open('add_params.xml', 'rb') as f:
    data = f.read()

response = requests.post(
    'http://localhost:28139/api/2.0/fo/auth/unix/',
    params=params,
    headers=headers,
    data=data,
    auth=('USER', 'PASS'),
)

File: test/fixtures/r/bash_redirect.r
library(httr)

headers = c(
  `X-Requested-With` = "curl",
  `Content-Type` = "text/xml"
)

params = list(
  action = "create",
  title = "UnixRecord",
  username = "root",
  password = "abc123",
  ips = "10.10.10.10"
)

data = upload_file("add_params.xml")
res <- httr::POST(url = "http://localhost:28139/api/2.0/fo/auth/unix/", httr::add_headers(.headers=headers), query = params, body = data, httr::authenticate("USER", "PASS"))
